docs/prober-and-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Prober and monitoring
+
+**PMAP is not a official Google product.**
+
+Monitoring consists of two parts: prober and alerting.
+
+## Prober
+
+[Prober](../prober/) is a go binary being deployed to [cloud run
+job](https://cloud.google.com/run/docs/overview/what-is-cloud-run#jobs), and is
+triggered by [cloud scheduler](https://cloud.google.com/scheduler) to constantly
+probing pmap services to check if the services are up.
+
+In each prober job, the prober will upload a yaml file to
+[gcs](https://cloud.google.com/storage), and query the bigquery table to see if
+a corrosponding entry exists in bigquery table. The job is considered as success
+on if the following requirements for both mapping and policy service are met:
+
+1. The object is uploaded.
+2. There is an entry in bigquery table that matches the traceID.
+
+## Monitoring and Alert Policies
+
+We monitor the pmap service with [native cloud run monitoring
+metrics](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-run) and set up
```

Comment:
Which metrics do we use here for cloud run, do you mean the `job/completed_execution_count`?